File: app/src/main/java/com/babestudios/companieshouse/data/model/insolvency/InsolvencyCase.java
package com.babestudios.companieshouse.data.model.insolvency;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class InsolvencyCase {
	@SerializedName("dates")
	public List<Date> dates = new ArrayList<>();
	@SerializedName("links")
	public InsolvencyCaseLinks links;
	@SerializedName("notes")
	public List<String> notes = new ArrayList<>();
	@SerializedName("number")
	public String number;
	@SerializedName("practitioners")
	public List<Practitioner> practitioners = new ArrayList<>();
	@SerializedName("type")
	public String type;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/insolvency/InsolvencyCase.kt
package com.babestudios.companyinfouk.data.model.insolvency

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class InsolvencyCase {
	@SerializedName("dates")
	var dates: List<Date> = ArrayList()
	@SerializedName("links")
	var links: InsolvencyCaseLinks? = null
	@SerializedName("notes")
	var notes: List<String> = ArrayList()
	@SerializedName("number")
	var number: String? = null
	@SerializedName("practitioners")
	var practitioners: List<Practitioner> = ArrayList()
	@SerializedName("type")
	var type: String? = null
}
